go/saltpack/basic/key.go

```diff
@@ -5,6 +5,8 @@ package basic
 
 import (
 	"crypto/rand"
+	"reflect"
```

Comment:
I wasn't planning on anyone having to us this keyring, I more or less just included it as an example (which is why there no support for signing keyrings, etc.)